source/api/content-state/0.9/index.md

```diff
@@ -494,7 +374,7 @@ The client allows the content state URI or data to be pasted into part of its UI
 </script>
```

Comment:
Do we need to explain the use of text/plain here? we do down below for drag and drop